lib/DDG/Spice/BiblioSearch/BooksByAuthor.pm

```diff
@@ -0,0 +1,53 @@
+package DDG::Spice::BiblioSearch::BooksByAuthor;
+# ABSTRACT: This module fetches books authored by a person searched for in DDG.
+
+use strict;
+use DDG::Spice;
+use URI::Escape;
+
+# Caching - https://duck.co/duckduckhack/spice_advanced_backend#caching-api-responses
+spice is_cached => 1;
+
+# Metadata.  See https://duck.co/duckduckhack/metadata for help in filling out this section.
+name "BiblioSearch - Books By Author";
+source "Goodreads";
+icon_url "https://www.goodreads.com/images/icons/goodreads_icon_32x32.png";
+description "Displays a list of books written/co-writen/edited by certain author.";
+primary_example_queries "Books by Arundhati Roy, Books authored by Shakespeare";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/BiblioSearch/BooksByAuthor.pm";
+attribution github => ["iammrigank", "Mriganka Ghosh"],
+            twitter => "tweetmrigank";
+
+# Goodreads API endpoint
+my $api_key = '{{ENV{DDG_SPICE_BIBLIOSEARCH_GODREADS_APIKEY}}}';
```

Comment:
@iammrigank Are you pushing to the correct branch? It looks like these changes aren't taking effect.